### DISCUSSION TITLE
Image from weathercode

### DISCUSSION THREAD

**Discussion by arpa42** [open-meteo/open-meteo#217] [Q&A]
Does anyone known where to find image urls from the weathercode ?
It's for a Grafana integration.

**Comment by phillipseamore** [comment]
At the bottom of https://open-meteo.com/en/docs you'll find definitions of the codes. You can use any weather icon with the same definition, e.g. an icon library with a partlyclody.png = code 2

**Reply by arpa42** [reply]
Thanks for your answer, but I had already seen the definition of the codes. I am looking for an icon library that can be used from the codes or descriptions.

**Reply by patrick-zippenfenig** [reply]
Here are some nicely animated weather icons: https://github.com/basmilius/weather-icons

There are multiple weather icon repositories on GitHub that use WMO weather representation codes.

**Reply by arpa42** [reply]
Nice icons indeed ! 
I found https://github.com/erikflowers/weather-icons that use WMO codes, but it needs to be embedded in a project, and it's not possible with Grafana
